GOVERNANCE.md

```diff
@@ -65,109 +66,109 @@ including:
 * Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-* [Current list of CTC members](./README.md#current-project-team-members)
+* [Current list of TSC members](./README.md#current-project-team-members)
 
-## CTC Membership
+## TSC Membership
 
-CTC seats are not time-limited. There is no fixed size of the CTC. The CTC
+TSC seats are not time-limited. There is no fixed size of the TSC. The TSC
 should be of such a size as to ensure adequate coverage of important areas of
 expertise balanced with the ability to make decisions efficiently.
 
-There is no specific set of requirements or qualifications for CTC
+There is no specific set of requirements or qualifications for TSC
 membership beyond these rules.
 
-The CTC may add additional members to the CTC by a standard CTC motion.
+The TSC may add additional members to the TSC by a standard TSC motion.
 
-When a CTC member's participation in [CTC activities](#ctc-activities) has
-become minimal for a sustained period of time, the CTC will request that the
+When a TSC member's participation in [TSC activities](#tsc-activities) has
+become minimal for a sustained period of time, the TSC will request that the
 member either indicate an intention to increase participation or voluntarily
-resign.
+resign in accordance to the TSC Charter.
 
-CTC members may only be removed by voluntary resignation or through a standard
-CTC motion.
+TSC members may be removed by voluntary resignation, through a standard TSC
+motion, or through non-participation.
 
-Changes to CTC membership should be posted in the agenda, and may be
-suggested as any other agenda item (see [CTC Meetings](#ctc-meetings) below).
+Changes to TSC membership should be posted in the agenda, and may be
+suggested as any other agenda item (see [TSC Meetings](#tsc-meetings) below).
 
-No more than 1/3 of the CTC members may be affiliated with the same
-employer.  If removal or resignation of a CTC member, or a change of
-employment by a CTC member, creates a situation where more than 1/3 of
-the CTC membership shares an employer, then the situation must be
-immediately remedied by the resignation or removal of one or more CTC
+No more than 1/3 of the TSC members may be affiliated with the same
```

Comment:
Perhaps just update to 1/4 everywhere, if this fits with the current members set?